src/sentry/scim/endpoints/teams.py

```diff
@@ -430,6 +436,8 @@ def delete(self, request: Request, organization, team) -> Response:
         """
         Delete a team with a SCIM Group DELETE Request.
         """
+        request = self.fix_log_name(request)
```

Comment:
I'm curious whether there is a place upstream we could do this, or some kind of mixin we could add to the affected endpoint classes. It seems a little brittle to drop `self.fix_log_name(request)` in a lot of spots -- easy to miss one.

Reply:
there definitely is, I've moved it to create_audit_entry to reduce the number of times I have to do this to 1